bos2cob_py3.py

```diff
@@ -1452,7 +1452,9 @@ def preprocess(self):
 			return self.output.getvalue()
 
 		def on_error(self, file, line, msg):
-			print(file, line, msg)
+			print("File: " + file)

```

Comment:
File can be None, as seen from the previous error. 
I would recommend python's f-strings
`print(f"File: {file}: Line:{line}: Error:{msg}")`

Reply:
`"string" + None ` would throw another error otherwise